lib/attrsets.nix

```diff
@@ -199,16 +231,27 @@ rec {
      Example:
        attrVals ["a" "b" "c"] as
        => [as.a as.b as.c]
+
+     Type:
+       attrVals :: [String] -> AttrSet -> [Any]
   */
-  attrVals = nameList: set: map (x: set.${x}) nameList;
+  attrVals =
+    # The list of attributes to fetch from `set`. Each attribute name must exist on the attrbitue set
+    nameList:
+    # The set to get attribute values from
+    set: map (x: set.${x}) nameList;
 
 
   /* Return the values of all attributes in the given set, sorted by
      attribute name.
 
+     Provides a backwards-compatible interface of `builtins.attrValues` for Nix version older than 1.8.
+
```

Comment:
We don't need to include this in the docs, if anything the compatibility code can be removed since 1.8 is already way older than `lib/minver.nix`
```suggestion
```